### ISSUE TITLE
How to specify the monitor?

### ISSUE THREAD

**Issue by cctry** [Nonary/ResolutionAutomation#85]
I am using a virtual display in my setup, and the script attempts to set a specific resolution but fails. It is possibly because it tries to apply the resolution to the physical monitor, which does not support the requested resolution.

Is there a way to explicitly specify the monitor (virtual display) for the script?

**Comment by batkison** [comment]
having the same issue here with the dummy plug

**Comment by Nonary** [comment]
No, it only changes the primary monitor. Combine with the MonitorSwapper script or use the built in feature in Sunshine now (in latest release)

https://github.com/Nonary/MonitorSwapAutomation
